plugins/importexport/native/filter/NativeXmlPKPAuthorFilter.php

```diff
@@ -106,13 +106,6 @@ public function handleElement($node)
                         }
                         $author->setFamilyName($n->textContent, $locale);
                         break;
-                    case 'affiliation':
-                        $locale = $n->getAttribute('locale');
-                        if (empty($locale)) {
-                            $locale = $publication->getData('locale');
-                        }
-                        $author->setAffiliation($n->textContent, $locale);
-                        break;
```

Comment:
I believe this should stay, and the new element rorAffiliation needs to be considered as well, no?
Maybe the only thing is: if we have the affiliation element, if we would like to see if the ROR exists for it, like in the upgrade script :thinking: